source/locale/en/symbols.dic

```diff
@@ -117,22 +112,18 @@ _	line	most
 ◾	black square	some
 □	white square	some
 ◦	white bullet	some
-⇒	right double arrow	some
 ⇨	right white arrow	some
 ➔	right-pointing arrow	some
 ➢	right arrowhead	some
-⮚	right arrowhead	some

```

Comment:
was this removed intentionally?

Reply:
Yes, the entry was duplicate, see line above.